msu/hooks/skills/skill.nut

```diff
@@ -386,12 +384,11 @@
 		return this.onVerifyTarget(_origin, _targetTile) && this.isInRange(_targetTile, _origin);
 	}
 
-	local getDescription = o.getDescription;
-	o.getDescription = function()
+	q.getDescription = @(__original) function()
 	{
-		if (this.m.DamageType.len() == 0 || !::MSU.Mod.ModSettings.getSetting("ExpandedSkillTooltips").getValue())
+		if (this.m.DamageType == null || this.m.DamageType.len() == 0 || !::MSU.Mod.ModSettings.getSetting("ExpandedSkillTooltips").getValue())
```

Comment:
This seems like #291 and therefore now sure what it's doing here

Reply:
As prev